src/axom/quest/examples/CMakeLists.txt

```diff
@@ -263,6 +263,30 @@ if(AXOM_ENABLE_MPI AND MFEM_FOUND AND MFEM_USE_MPI
             unset(_policies)
         endif()
 
+        # Test 2D MFEM mesh (linearize contours)
+        set(_testname quest_shaping_driver_ex_heroic_roses_mfem)
+        axom_add_test(
+            NAME ${_testname}
+            COMMAND  quest_shaping_driver_ex 
+                     -i ${shaping_data_dir}/heroic_roses_mfem.yaml
+                     --method sampling
+                     inline_mesh --min 0 0 --max 300 400 --res 300 400 -d 2
+            NUM_MPI_TASKS ${_nranks})
+        set_tests_properties(${_testname} PROPERTIES
+            PASS_REGULAR_EXPRESSION  "Volume of material 'black' is 36,637.")
+
+        # Test 2D MFEM mesh (winding number)
+        set(_testname quest_shaping_driver_ex_heroic_roses_mfem_cp)
+        axom_add_test(
+            NAME ${_testname}
+            COMMAND  quest_shaping_driver_ex 
+                     -i ${shaping_data_dir}/heroic_roses_mfem_cp.yaml
+                     --method sampling --sampling windingnumber
+                     inline_mesh --min 0 0 --max 300 400 --res 300 400 -d 2
+            NUM_MPI_TASKS ${_nranks})
+        set_tests_properties(${_testname} PROPERTIES
+            PASS_REGULAR_EXPRESSION  "Volume of material 'black' is 36,645.")
```

Comment:
Presumably the volumes are different b/c the winding numbers are on the curved geometry and the inout sampling is discretized.